src/Apps/Settings/Routes/MyCollection/Components/MyCollectionEmptyState.tsx

```diff
@@ -7,230 +7,150 @@ import {
   Image,
   Message,
   ResponsiveBox,
+  Spacer,
   Text,
 } from "@artsy/palette"
 import { useMyCollectionTracking } from "Apps/MyCollection/Routes/Hooks/useMyCollectionTracking"
-import { EmptyMyCollectionPageProps } from "Apps/Settings/Routes/MyCollection/Components/EmptyMyCollectionPage"
+import { useAuthDialog } from "Components/AuthDialog"
 import { ModalType } from "Components/Authentication/Types"
 import { RouterLink } from "System/Router/RouterLink"
 import { useSystemContext } from "System/SystemContext"
 import { useFeatureFlag } from "System/useFeatureFlag"
-import { openAuthModal } from "Utils/openAuthModal"
 import { resized } from "Utils/resized"
-import { Media } from "Utils/Responsive"
 
-const DesktopLayout: React.FC<EmptyMyCollectionPageProps> = ({
-  loggedOutState,
-}) => {
+export const MyCollectionEmptyState: React.FC = () => {
   const isCollectorProfileEnabled = useFeatureFlag("cx-collector-profile")
 
-  const { mediator } = useSystemContext()
+  const { isLoggedIn } = useSystemContext()
+
   const {
     addCollectedArtwork: trackAddCollectedArtwork,
   } = useMyCollectionTracking()
 
   const image = resized(
     "https://files.artsy.net/images/my-coll-get-app-img.jpg",
-    {
-      width: 770,
-      height: 652,
-    }
+    { width: 910, height: 770 } // TODO: Source image should be 4:3, ideally. Is currently 709:600
```

Comment:
This image was being sized too small and I would work with design on fixing this to a standard aspect ratio.